mediaqueries-5/Overview.bs

```diff
@@ -478,6 +478,36 @@ Detecting the desire for less motion on the page: the 'prefers-reduced-motion' f
 
 	</dl>
 
+<h3 id="prefers-reduced-transparency">
+Detecting the desire to reduce the use of transparent or translucent layering effects on the page: the 'prefers-reduced-transparency' feature</h3>
```

Comment:
> Transparency is a boolean concept; translucency and opacity are not.

I'm affraid I'll discagree on this one. I check the dictionary, and it gave not hit of this nuance. https://encrypted.google.com/search?q=partially+transparent&tbm=isch gives plenty of relevant results. Apple used this phrasing in their System Preferences to express the same concept. Using the word transparency is fine.